src/containers/Search/CloneModel.js

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadMyProjectsAction } from 'store/actions/project';
+import { clonePlaylist, cloneActivity } from 'store/actions/search';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Accordion, Card, Button } from 'react-bootstrap';
+import Swal from 'sweetalert2';
+
+export default function LTIProjectShared(props) {
+  const [activeProject, setActiveProject] = useState('');
+  const [activePlaylist, setActivePlaylist] = useState('');
+  // const [activeactivity, setactiveactivity] = useState("");
+  // const [activeactivitydefault, setactiveactivitydefault] = useState(true);
+  const [allproject, setAllproject] = useState(null);
+  const [currentProject, setCurrentProject] = useState();
+  const [currentPlaylist, setCurrentPlaylist] = useState();
+  const dispatch = useDispatch();
+  const allProjects = useSelector((state) => state);
+  useEffect(() => {
+    dispatch(
+      loadMyProjectsAction(),
+    );
+  }, [dispatch]);
+
+  useEffect(() => {
+    allProjects.project.projects && setAllproject(allProjects.project.projects);
+  }, [allProjects]);
+  return (
+    <div className="ltiallproject">
+      <Accordion className="top-box-project">
+        {!!allproject
+          && allproject.map((data, counter_top) => (
+            <Card>
+              <Card.Header className="topcard">
+                <Accordion.Toggle
+                  as={Button}
+                  variant="link"
+                  eventKey={counter_top + 1}
+                  className="topcardbtn"
+                >
+                  <span
+                    onClick={() => {
+                      if (activeProject === counter_top + 1) {
+                        setActiveProject();
+                      } else {
+                        setActiveProject(counter_top + 1);
+                        setCurrentProject(data.id);
+                        setCurrentPlaylist();
+                        // setactiveactivity();
+                      }
+                      // setActivePlaylist();
+                    }}
+                  >
+                    <div className="flex-bar">
+                      <div>
+                        {activeProject === counter_top + 1 ? (
+
+                          <FontAwesomeIcon icon="check-square" />
```

Comment:
class mr-2 need

Reply:
no need here. it is being cared through css already